pkg/operator/ceph/cluster/rbd/spec.go

```diff
@@ -101,6 +101,11 @@ func (m *Mirroring) makeMirroringDaemonContainer(daemonConfig *daemonConfig) v1.
 		Env:             controller.DaemonEnvVars(m.cephVersion.Image),
 		Resources:       m.resources,
 		SecurityContext: mon.PodSecurityContext(),
+		// TODO:
+		// Not implemented at this point since the socket name is '/run/ceph/ceph-client.rbd-mirror.a.1.94362516231272.asok'
+		// Also the command to run will be:
+		// ceph --admin-daemon /run/ceph/ceph-client.rbd-mirror.a.1.94362516231272.asok rbd mirror status
+		// LivenessProbe:   controller.GenerateLivenessProbeExecDaemon(string(config.RbdMirrorType), daemonConfig.DaemonID),
```

Comment:
What's preventing this from being implemented? We need a change from ceph?

Reply:
It's just more changes and the timeframe to 1.3 is short so I prefer to leave this for another time. Also given the low adoption it's not critical to have it now.

Reply:
I think it plays well with the commit message in that case especially when browsing the code.  It's not uncommon to leave some TODO in the code. I think it makes sense to have it. I'll come back to it at some point :)

Reply:
What about opening a github issue for it? Then we keep better track of it. Leaving the comments is fine as long as it's a planned fix.